charts/headlamp/values.yaml

```diff
@@ -84,7 +84,7 @@ config:
   # -- directory to look for plugins
   pluginsDir: "/headlamp/plugins"
   # Extra arguments that can be given to the container. See charts/headlamp/README.md for more information.
-  extraArgs: {}
+  extraArgs: []
```

Comment:
This doesn't seem related to the patch.

Reply:
Fixed in a seperate commit. extraArgs should indeed be an array rather than an object. Since array can also contain object values.